kernel/fmt.cc

```diff
@@ -326,6 +326,16 @@ void Fmt::parse_verilog(const std::vector<VerilogFmtArg> &args, bool sformat_lik
 				break;
 			}
 
+			case VerilogFmtArg::TIME: {
+				FmtPart part = {};
+				part.type = FmtPart::TIME;
+				part.realtime = arg->realtime;
+				part.padding = ' ';
+				part.width = 20;
```

Comment:
I copied what `$display("%t", $time)` does, not sure up to spec

Reply:
I think that's not up to spec. I'm not sure who introduced this functionality--I think that was after I offered the fmt PR.